test/performance/benchmarks/broker-pubsub/cluster.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright 2019 The Knative Authors
```

Comment:
update copyright, we are using `Copyright 2019 Google LLC` in this repo... 
here an in the other files as well

Reply:
Done.